app/assets/stylesheets/static_pages.css.scss

```diff
@@ -1,3 +1,69 @@
 // Place all the styles related to the static_pages controller here.
 // They will automatically be included in application.css.
 // You can use Sass (SCSS) here: http://sass-lang.com/
+
+// Footer
+.full-width {
+  max-width: 1600px;
+}
+
+.tagline {
+  padding-left: 1rem;
+}
+
+.login-box {
+  margin-top: 0.75rem;
+  p {
+    margin-bottom: 0;
+  }
+}
+
+/* Styles for signup panels */
+.auth-plain {
+  padding-right: 0px;
+  padding-left: 0px;
+  padding-top: 0px;
+}
+
+.left-solid {
+margin-top: 0px;
+margin-bottom: 0px;
+}
+
+.signup-panel {
+  border-radius: 5px;
+  padding: 15px;
+  margin-top: 30px;
+  margin-bottom: 30px;
+  background: #fff;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.